clikt/build.gradle.kts

```
@@ -21,6 +21,14 @@ buildscript {
 
 kotlin {
     jvm()
+
+    /**
```

This shouldn't be a KDoc style comment because it's not a declaration but a function call.
```suggestion
    /*
```